### PR TITLE
Support transparancy for the _background image processor's attribute

### PR DESCRIPTION
**Pull request by theus77**
if the _background asset config attribute is set to #FF0000 or #FF000000, the background will be red without transparency. With #FF0000FF it will be fully transparent. It works only if the _quality is set to 0 and than if a PNG is generated

### CODE REVIEW

File: Storage/Processor/Image.php [hunk 1]
```diff
@@ -97,6 +97,21 @@ private function getWidthHeight(array $size): array
         return [$width, $height];
     }
 
+    private function fillBackgroundColor($temp)
+    {
+        $background = $this->config->getBackground();
+        imagesavealpha($temp, true);
+
+        $solid_colour = imagecolorallocatealpha(
```
Comment (dameert) [inline Storage/Processor/Image.php]: No issues with PHPcs in this _ variable? Should be camelCase

File: Storage/Processor/Image.php [hunk 2]
```diff
@@ -38,7 +38,7 @@ public function generate(string $filename)
             $image = $this->applyBackground($image, $width, $height);
         }
 
-        if (null !== $this->config->getRadius()) {
+        if (($this->config->getRadius() ?? 0) > 0) {
```
Comment (dameert) [inline Storage/Processor/Image.php]: This kind of nesting in the if test makes readability harder. Why not return 0 in getRadius by default in stead of null? Null would never make sense for a radius.

File: Storage/Processor/Image.php (hunk 1 above)
Comment (theus77) [reply Storage/Processor/Image.php]: Agree, but I didn’t face issue word phpcs. I’ll fit it

Comment (theus77) [reply Storage/Processor/Image.php]: Solved

File: Storage/Processor/Image.php (hunk 2 above)
Comment (theus77) [reply Storage/Processor/Image.php]: Agree. Solved

File: Storage/Processor/Image.php [hunk 3]
```diff
@@ -47,7 +47,7 @@ public function generate(string $filename)
         }
 
         $path = tempnam(sys_get_temp_dir(), 'ems_image');
-        if ($this->config->getQuality()) {
+        if (($this->config->getQuality() ?? 0)  > 0) {
```
Comment (dameert) [inline Storage/Processor/Image.php]: I think the same goes for quality? Should return 0 by default. (I don’t really like this combination of ?? Inside an if test. This is making 0 a default value so probably getQuality should by default return 0? If not it should by default generate 100, but not an incompatible value (null)

Comment (theus77) [reply Storage/Processor/Image.php]: Fixed

**Pull request merged**